addons/point_of_sale/static/src/js/models.js

```diff
@@ -3250,6 +3250,16 @@ exports.Order = Backbone.Model.extend({
     is_empty: function(){
         return this.orderlines.models.length === 0;
     },
+    /**
+     * If the order is empty (there are no products)
+     * and all "pay_later" payments are negative,
+     * we are settling a customer's account
+     * @returns {boolean} true if the current order is a settlement, else false
+     */
+    is_settlement: function() {
```

Comment:
Sure I can, I just thought that since it was a relatively generic concept, it could benefit from being in the general model, unless if you have other criteria for a settlement in general.
But if you prefer I can move it to the localisation, no problem. (waiting for your confirmation)